### ISSUE TITLE
unclear function selector in call to purity checker

### ISSUE THREAD

**Issue by ralexstokes** [ethereum/casper#94]
https://github.com/ethereum/casper/blob/b2a1189506710c37bbdbbf3dc79ff383dbe13875/casper/contracts/simple_casper.v.py#L298

Looks like a typo...

The `raw_call` presumably calls the `submit` function here: https://github.com/ethereum/research/blob/master/impurity/check_for_impurity.se

And following our convention we see:
```bash
> web3.sha3("submit(address)")
"0xa1903eab65b303e5d2b0f7ffc455cbdb5d081456ef23967ae426c2cada7ffffd"
```

which implies the method selector should be `'\xa1\x90\x3e\xab'`.

If someone can confirm I'll submit a PR.

**Comment by djrtwo** [comment]
`"\x3e" == ">"`

I agree that it would be cleaner with the explicit hex if you still want to do that PR

**Comment by djrtwo** [comment]
addressed in #97
